### PR TITLE
Add suggestions in filter textfield

### LINKED ISSUE #94: Request: Adding suggestions in the filter textfield
We can enable suggestions in the filter text field to simplify filtering resources by name.

```go
// from bubbels.Textfield
func (m *Model) SetSuggestions(suggestions []string) {...}
```

### PR DESCRIPTION
**Pull request by semihbkgr**
fixes #94

### CODE REVIEW

**Review by applejag — CHANGES_REQUESTED** [review]
Another great contribution! Thanks @semihbkgr 

I just had a few comments

**Comment by applejag** [comment]
Don't worry about the goreleaser action failing. It's because this is a fork, and I haven't configured the workflow to work on forks. My bad

File: pkg/table/row.go [hunk 1]
```diff
@@ -59,10 +59,11 @@ func (c AgoColumn) String() string {
...
 type Row struct {
-	ID        string
-	Fields    []any
-	Status    Status
-	SortField string
+	ID          string
+	Fields      []any
+	Status      Status
+	SortField   string
+	FilterField string
```
[the remark this answers is not in the document]
Comment (semihbkgr) [reply pkg/table/row.go:66]: Agreed 👍
better to use `...Key` for the values not displayed in the table.

**Comment by applejag** [comment]
> Don't worry about the goreleaser action failing. It's because this is a fork, and I haven't configured the workflow to work on forks. My bad

Hmm it was actually something wrong with snapcraft, not my setup. It works fine now that they've published a patch.

But then the govulncheck fails because there was a new version of Go :sweat_smile: 

I pushed some changes to your PR. I hope you don't mind!

**Review by applejag — APPROVED** [review]

**Pull request merged**